prov/usnic/src/usdf.h

```diff
@@ -177,7 +178,10 @@ struct usdf_pep {
 	struct usdf_fabric *pep_fabric;
 	struct usdf_eq *pep_eq;
 	int pep_sock;
-	struct sockaddr_in pep_src_addr;
+	union {
```

Comment:
Because pep structure is the only structure that actually store the information in itself. (via memcpy) I want to preserve the behavior.